src/games/racing/racing.py

```diff
@@ -1,6 +1,116 @@
+from random import randrange
+
+import pygame
+from pygame import *
+
 from src.games.game import *
+from src.games.racing.board import Board
+from src.games.racing.enemy import Enemy
+from src.games.racing.player import Player
+from src.games.racing.settings import Settings as stgns
+from src.resources.layout_rsc import LayoutRsc
+from src.resources.racing_resources import RacingResources
 
 
 class Racing(Game):
+    CRASH_SOUND = pygame.mixer.Sound(RacingResources.CRASH_SOUND_PATH)
+
     def __init__(self, level, is_sound_on):
         Game.__init__(self, level, is_sound_on)
+        self._board = Board(LayoutRsc.GAME_AREA_WIDTH, LayoutRsc.GAME_AREA_HEIGHT, * stgns.BOARD_FIELDS)
+        self._player = Player(self._board)
+        self._group_of_enemies = sprite.Group()
+        self._refresh_time = stgns.LVL_TO_REFRESH_TIME_MAP[level]
+        self._time_since_last_update = self._refresh_time
+        self._level = level
+        self._points_as_float = 0.0
+        self._empty_buffer_rect = Rect((0, -stgns.CAR_SIZE[1]),
+                                       (LayoutRsc.GAME_AREA_WIDTH, 2 * stgns.CAR_SIZE[1]))
+        self._points_earned_per_update = 1 / 2 * stgns.GAME_SPEEDS_SCORE_BONUS[self._level]
+        self._is_running = True
+        self._lives = stgns.AMOUNT_OF_LIVES
+
+    def process_events(self, joystick):
+        for event in pygame.event.get():
+            joystick.process_event(event)
+            if event.type in {JOYBUTTONUP, JOYBUTTONDOWN, JOYAXISMOTION}:
+                if joystick.is_arrow_leftdir_pressed():
+                    self._player.take_roadway("left")
+                if joystick.is_arrow_rightdir_pressed():
+                    self._player.take_roadway("right")
+
+    def update(self, delta_time):
+        self._time_since_last_update -= delta_time
+        if self._time_since_last_update <= 0:
+
+            if self._has_player_collided():
+                self._die()
+
+            if self._lives == -1:
+                self._is_running = False
+
+            self._player.update()
+            self._create_enemy_if_possible()
+            self._group_of_enemies.update()
+            self._time_since_last_update = self._refresh_time
+            self._points_as_float += self._points_earned_per_update
+            self._points = int(self._points_as_float)
```

Comment:
That's the only place where you use self._points_as_float. Why can't you do:
```suggestion
            self._points = int(self._points_earned_per_update)
```